packages/data-stores/src/site/actions.ts

```diff
@@ -272,11 +272,11 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		yield saveSiteSettings( siteId, { blogdescription } );
 	}
 
-	function* setThemeOnSite( siteSlug: string, theme: string ) {
+	function* setThemeOnSite( siteSlug: string, theme: string, styleVariation?: string ) {
 		yield wpcomRequest( {
 			path: `/sites/${ siteSlug }/themes/mine`,
 			apiVersion: '1.1',
-			body: { theme: theme, dont_change_homepage: true },
+			body: { theme: theme, style_variation: styleVariation || '', dont_change_homepage: true },
```

Comment:
Nit: I wonder why we need to coalesce `styleVariation` with `''`, why not leave it `undefined`/`null`? I think the backend code checks with `empty()` which works with `null` / empty string anyway. So the code can be simplified to:

```
body: { theme, style_variation, dont_change_homepage: true }
```

Reply:
IIRC, without it I'm getting `undefined` as string, I will double-check in a bit. If that is the case, I think it is probably ok to also set `''` as the default value of the param.

Reply:
> IIRC, without it I'm getting `undefined` as string, I will double-check in a bit. 

That is no longer the case when I retested. So removing the empty string fallback in https://github.com/Automattic/wp-calypso/pull/67141/commits/d941d00d39a77310590eba69525da110bff0ffed. When `styleVariation` is not passed in, the prop is not assigned at all, which is good.

Keep one dashed-case and one camelCased though just because of code standards between JS & PHP. `style_variation: styleVariation`.